package.json

```diff
@@ -82,6 +82,7 @@
   "volta": {
     "node": "18.16.1",
     "yarn": "1.22.19",
-    "npm": "9.8.0"
+    "npm": "9.8.0",
+    "pnpm": "7.32.5"
```

Comment:
I think we can probably have only `node` and `pnpm`, as it looks like we aren't using `yarn` or `npm` directly here

Reply:
updated